plugin/src/main/java/org/opensearch/ml/stats/MLNodeLevelStat.java

```diff
@@ -11,7 +11,8 @@
  */
 public enum MLNodeLevelStat {
     ML_JVM_HEAP_USAGE,
-    ML_EXECUTING_TASK_COUNT,
+    ML_EXECUTING_TASK_COUNT, // How many tasks are executing currently. If any task starts, then it will be 1, if the task finished then it
```

Comment:
`then it will be 1` -> `then it will increase by 1`

`will get back to 0` -> `will decrease by 1`?